src/Storage/Storage/Blob/Cmdlet/SetAzureStorageBlobContent.cs

```diff
@@ -170,6 +171,30 @@ public PremiumPageBlobTier PremiumPageBlobTier
 
         private PremiumPageBlobTier? pageBlobTier = null;
 
+        [Parameter(HelpMessage = "Block Blob Tier, valid values are Hot/Cool/Archive. See detail in https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-storage-tiers", Mandatory = false)]
+        [PSArgumentCompleter("Hot", "Cool", "Archive")]
+        [ValidateNotNullOrEmpty]
+        public string StandardBlobTier
```

Comment:
Should we keep `[ValidateSet("Hot", "Cool", "Archive", IgnoreCase = true)]` since completer doesn't provide validation functionality? Otherwise Enum.Parse may throw exception.

Reply:
ValidateSet added back, 
ValidateNotNullOrEmpty is removed, since null and empty string will fail with ValidateSet  added.